actions/post_actions.ts

```diff
@@ -266,7 +266,7 @@ export function markPostAsUnread(post: Post, location: string) {
             const threadId = post.root_id || post.id;
             ThreadActions.handleFollowChanged(dispatch, threadId, currentTeamId, true);
             dispatch(manuallyMarkThreadAsUnread(threadId, post.create_at - 1));
-            await dispatch(ThreadActions.markThreadAsUnread(userId, currentTeamId, threadId, post.id));
+            await dispatch(ThreadActions.markThreadAsUnread(userId, currentTeamId, threadId));
```

Comment:
why do we remove the post id?

Reply:
As we are getting posts inside threads action, there is no need to pass `post.id` to indicate which post is the last post before mark as read

https://github.com/mattermost/mattermost-webapp/blob/e032a60256b4328099156d183712a387a08f221c/packages/mattermost-redux/src/actions/threads.ts#L266